boot/seal.go

```diff
@@ -71,11 +71,9 @@ const (
 
 // FDESetupHookParams contains the inputs for the fde-setup hook
 type FDESetupHookParams struct {
-	Key     secboot.EncryptionKey
+	Key     []byte
 	KeyName string
```

Comment:
This is a bit misleading, secboot here referes to "our" github.com/snapcore/snapd/secboot that wraps around your secboot. We did it his way so that we can build on platforms like debian/fedora where we cannot vendor code and where no secboot package to build against is available.

Of course we could "copy" KeyPayload into our secboot but it seems just using []byte here is ok?